.travis.yml

```diff
@@ -32,55 +33,11 @@ jobs:
         - travis_retry curl -L https://get.haskellstack.org/stable/linux-x86_64.tar.gz | tar xz --wildcards --strip-components=1 -C ~/.local/bin '*/stack'
         - stack setup
     - stage: Build dependencies
-      name: "root"
+      name: "Build dependencies"
       script:
+        # - stack clean --full  # if unreliability persists
```

Comment:
a.f.a.i.k. dependencies are stored in `~/.stack`, and this does nothing to fix dependency errors